src/content/lib/CreateJobContent/CreateJobForm.js

```diff
@@ -35,8 +36,56 @@ class CreateJobForm extends React.Component {
   handleChangeHourlyRating = (event, index, value) => this.setState({ hourlyRatingValue: value })
   // handleChangeState = (event, index, value) => this.setState({ stateValue: value })
 
+  handleBack () {
+    Router.pushRoute('/opportunities')
+  }
+
+  renderBudgetWidget = () => {
+    if (!this.props.flowType) return null
+    return this.props.flowType === 1 ? (
```

Comment:
do not use magic numbers
use predefined constants
like
JOB_FLOW_TYPE = { FIXED_PRICE: 2, FIXED_HOURS: 1 }

return this.props.flowType === JOB_FLOW_TYPE.FIXED_HOURS

this code more readable